vm/systemSmartContracts/delegation.go

```diff
@@ -2645,6 +2658,19 @@ func (d *delegation) getMetaData(args *vmcommon.ContractCallInput) vmcommon.Retu
 	return vmcommon.Ok
 }
 
+func (d *delegation) addTokens(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
This method does not add anything, it just returns UserError or Ok

Reply:
yes. but it actually does - accepts eGLD.